Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -336,6 +336,7 @@ class BrowserViewController: UIViewController {
         }
         Preferences.NewTabPage.selectedCustomTheme.observe(from: self)
         // Lists need to be compiled before attempting tab restoration
+        ContentBlockerHelper.clearBlockingRules()
```

Comment:
@kylehickinson worked on this more, I will leave this part to him,
if you want me to take over and look closer at it, let me know Kyle